docs/UserGuide.md

```diff
@@ -131,52 +131,58 @@ Simply click on any command below to learn more about it!
 | [`edit INDEX ...`](#edit) | Edit a contact |
 | [`edit INDEX [t/TAG]`](#edit_tag) | Create / Edit tags for existing contacts |
 | [`edit INDEX t/`](#delete_tag) | Delete all tags from a contact |
-| [`edit INDEX [p/PLATFORM]`](#edit_platform) | Modify social media handles for a contact |
-| [`edit INDEX [date/NAME:YYYY-MM-DD[:monthly|:yearly]]…​`](#edit_dates) | Modify dates for a contact |
+| [`edit INDEX [p/PLATFORM]`](#edit_platform) | Modify social media handles of a contact |
+| [`edit INDEX [date/NAME:YYYY-MM-DD[:monthly|:yearly]]…​`](#edit_dates) | Modify associated dates of a contact |
```

Comment:
Just have to take note that this portion will be superseded by the updated representation for `edit` in #204 when merging is performed.